R/pkg/R/SQLContext.R

```diff
@@ -744,6 +747,9 @@ read.df.default <- function(path = NULL, source = NULL, schema = NULL, ...) {
   if (is.null(source)) {
     source <- getDefaultSqlSource()
   }
+  if (source == "csv" && is.null(options[["nullValue"]])) {
```

Comment:
Thanks for that pointer to the SQL code. I guess my point is that we could always pass the `nullValue` to the scala side and not do additional filtering on the R side for `source == csv` ?